src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    private String name;
+
+    public Truck(String name) {
+        this.name = name;
+    }

```

Comment:
Remove constructor and "name" field to be able to create instances without providing a name